cmd/githubgen/main.go

```diff
@@ -118,7 +118,8 @@ type DependabotData struct {
 
 func makePriority(status *Status) int {
 	// not an internal component such as pkg/**, and no distributions:
-	if len(status.Distributions) == 0 && status.Class != "" {
+	if (status.Class == "receiver" || status.Class == "processor" || status.Class == "exporter" || status.Class == "connector" || status.Class == "extension" || status.Class == "cmd") &&
```

Comment:
Shouldn't we allow `pkg` modules now that they can have stability levels?

Reply:
This if statement makes it so the priority of pkg (and any other future group like internal) have their priority set using the switch statement below and not this special case.  Without these checks, the pkg type was getting the lowest priority because it wasn't in any distributions.  This results in `make gendependabot` removing ottl from the dependabot list.

Reply:
Got it, thanks. Won't `cmd` modules hit the same case since they're also not in distributions?

Reply:
They don't have `class` set I believe.

Reply:
should gendependabot be affected by this change?

Reply:
It was bc gendependabot depends on mdatagen and I had to update mdatagen to handle generating a README for a pkg.  OTTL is now the first pkg to include a header in the readme and it broke some assumptions in mdatagen and githubgen.